src/main/scala/com/gu/restaurant_review_parser/ParsedRestaurantReview.scala

```diff
@@ -0,0 +1,49 @@
+package com.gu.restaurant_review_parser
+
+import java.time.OffsetDateTime
+
+case class WebAddress(value: String) extends AnyVal
```

Comment:
The slides (@cb372) and the code is missing the mandatory `val` to be value classes.
> which is marked with val and which has public accessibility